backend/internal/storage/postgres/schema/playlist.go

```diff
@@ -0,0 +1,59 @@
+package schema
+
+import (
+	"context"
+	"platnm/internal/models"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type PlaylistRepository struct {
+	*pgxpool.Pool
+}
+
+func (r *PlaylistRepository) CreatePlaylist(ctx context.Context, playlist models.Playlist) error {
+
+	query := `
+	INSERT INTO playlist (title, user_id, bio, cover_photo)
+	VALUES ($1, $2, $3, $4);
+	`
+
+	_, err := r.Exec(ctx, query, playlist.Title, playlist.UserID, playlist.Bio, playlist.CoverPhoto)
+
+	if err != nil {
+		return err
+	}
+	return nil
+
+}
+
+func (r *PlaylistRepository) AddToUserOnQueue(ctx context.Context, id string, track models.Track) error {
+
+	var playlistID string
+
+	findQuery := `SELECT id FROM playlist WHERE user_id = $1 AND title = 'On Queue'`
```

Comment:
this made me think of a tiny edge case where a user could potentially create their own playlist also called "On Queue" and mess this up

(no action items for you but we should discuss as a team if we wanna ban this as a custom playlist name or handle it some other way in the future)

Reply:
makes sense!